finalEX.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+import os
+from datetime import datetime
+from db import session, Upload
+from pptx_parser import read_presentation
+from gpt import generate_answer_gpt
+from utils import save_answer_on_json_file
+
+
+async def main():
+    """
+    Main function to execute the program.
+
+    This function continuously scans the uploads folder for pending files.
+    It processes each pending file by generating answers using GPT for each slide in the presentation.
+    The answers are then saved in a JSON file and the status of the file is updated to 'done'.
+    The processed files are deleted from the uploads folder.
+    """
+    while True:
+        asyncio.sleep(10)

```

Comment:
Also a constant.
That way it's much easier and quicker to config
I would even put it in a separate config file